CONTRIBUTING.md

```diff
@@ -11,11 +11,17 @@ ask on our mailing list opensuse-factory@opensuse.org
 It can be beneficial to learn how an openQA job is executed. To setup your own
 openQA instance, please refer to the documentation of the [openQA project](https://github.com/os-autoinst/openQA).
 
-Please, find up-to-date documentation references on the official [openQA project web-page](http://open.qa/documentation/).
+Please, find up-to-date documentation references on the official [openQA project web-page](https://open.qa/documentation/).
 
 If you are looking for a task to start with, check out the [openQA Tests](https://progress.opensuse.org/projects/openqatests/issues/)
 Redmine project. Look for tickets with [easy] or [easy-hack] tags.
 
+#### Relevant documentation
+
+* All openQA documentation in a single [html page](https://open.qa/docs/)
+* openQA [testapi](http://open.qa/api/testapi/) documentation
+* (wip) Available test library documentation http://os-autoinst.github.io/openQA/
```

Comment:
It's very likely that we either will forget to update this one or will never leave "wip" state. I suggest to simply delete the `(wip)` as it's the responsibility of a different repo to declare it "done" anyway.

Reply:
I prefer to keep it, everything is WIP, but this in particular, still needs a lot of progress.

> Can you verify that it looks better when converted with asciidoctor?

We don't use asciidoctor for this repository. However there was a mistake in my last change: https://github.com/foursixnine/os-autoinst-distri-opensuse/blob/docs-update/CONTRIBUTING.md now it should render properly